packages/api/migrations/20200820002013-create-nomination.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Nominations', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      status: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      dateReceived: {
+        type: Sequelize.DATE
+      },
+      providerFirstName: {
+        type: Sequelize.STRING
+      },
+      providerLastName: {
+        type: Sequelize.STRING
+      },
+      providerNumber: {
+        type: Sequelize.INTEGER
+      },
+      emailAddress: {
+        type: Sequelize.STRING
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      validated: {
+        type: Sequelize.BOOLEAN
+      },
+      hospitalName: {
+        type: Sequelize.STRING
+      },
+      hospitalURL: {
+        type: Sequelize.STRING
+      },
+      hospitalAddress: {
+        type: Sequelize.STRING
+      },
+      hospitalCity: {
+        type: Sequelize.STRING
+      },
+      hospitalState: {
+        type: Sequelize.STRING
+      },
+      hospitalZipCode: {
+        type: Sequelize.INTEGER
```

Comment:
While they're technically made up of numbers, ZIP codes don't really function like integers (you can't, for example, add and subtract them). What do you think about using string here?

Reply:
Makes sense!